spec/support/geckoboard_helpers.rb

```diff
@@ -39,4 +39,17 @@ def stub_geckoboard_dataset_find_or_create(dataset_id)
   def stub_geckoboard_dataset_post(dataset_id)
     stub_request(:post, "https://api.geckoboard.com/datasets/#{dataset_id}/data")
   end
+
+  def body_matching_geckoboard_data_for_claims(request, claims, date_field)
```

Comment:
This method returns `true` or `false` depending on the result of the comparison. Perhaps worth calling it something like `request_body_matches_geckoboard_data_for_claims?`.

Reply:
Yeah, nice. I was struggling with a good name for this method!